.teamcity/_self/PluginBaseBuild.kt

```diff
@@ -112,13 +112,11 @@ open class PluginBaseBuild : Template({
 				# These operations restore idempotence between the two builds.
 				rm -f ./release-archive/build_meta.txt
 
-				cd ./release-archive
 				%normalize_files%
-				cd ..
 
 				# 3. Check if the current build has changed, and if so, tag it for release.
 				# Note: we exclude asset changes because we only really care if the build files (JS/CSS) change. That file is basically just metadata.
-				if ! diff -rq --exclude="*.asset.php" $archiveDir ./release-archive/ ; then
+				if ! diff -ru --exclude="*.asset.php" --exclude="cache-buster.txt" $archiveDir ./release-archive/ ; then
```

Comment:
Could you elaborate on the flag change from `q` to `u` here?

Reply:
I'm planning to revert this, but using it to debug. `q` means that it won't print the actual diff of the file -- just whether or not the file has changed. `u` tells it to print using the unified diff format